.stickler.yml

```diff
@@ -0,0 +1,18 @@
+---
+linters:
```

Comment:
@frankchen07 Looks like you might have a bad rebase here.  These already exist in `master` 🤔